### PR TITLE
try to fix ui-blue background again

### PR DESCRIPTION
**Pull request by pbcomm**

### CODE REVIEW

**Comment by PeteSchuster** [comment]
// old
```
<svg xmlns="http://www.w3.org/2000/svg">
  <defs>
    <filter id="b" color-interpolation-filters="sRGB">
      <feGaussianBlur stdDeviation="55"/>
    </filter>
    <linearGradient id="a" gradientUnits="userSpaceOnUse" x1="550" y1="4" x2="550" y2="343">
      <stop offset="0" stop-color="#0066A0"/>
      <stop offset=".5" stop-color="#076699"/>
      <stop offset="1" stop-color="#003A74"/>
    </linearGradient>
  </defs>
  <path fill="url(#a)" d="M0 0h1100v343H0z"/>
  <g filter="url(#b)">
    <g fill="#0E74A0">
      <ellipse cx="72" cy="121" rx="95" ry="49"/>
      <ellipse cx="781" cy="171" rx="132" ry="166"/>
      <ellipse cx="1041" cy="231" rx="135" ry="160"/>
      <ellipse cx="541" cy="195" rx="132" ry="125"/>
    </g>
    <g fill="#0057A0">
      <ellipse cx="383" cy="23" rx="205" ry="136"/>
      <ellipse cx="278" cy="270" rx="83" ry="32"/>
      <ellipse cx="1083" cy="23" rx="117" ry="58"/>
    </g>
  </g>
</svg>
```


// new
```
<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="none">
  <defs>
    <filter id="a" color-interpolation-filters="sRGB">
      <feGaussianBlur stdDeviation="55"/>
    </filter>
  </defs>
  <g filter="url(#a)">
    <g fill="#0E74A0">
      <ellipse cx="72" cy="121" rx="95" ry="49"/>
      <ellipse cx="781" cy="171" rx="132" ry="166"/>
      <ellipse cx="1041" cy="231" rx="135" ry="280"/>
      <ellipse cx="541" cy="195" rx="132" ry="125"/>
    </g>
    <g fill="#0057A0">
      <ellipse cx="383" cy="23" rx="205" ry="136"/>
      <ellipse cx="278" cy="270" rx="83" ry="32"/>
      <ellipse cx="1083" cy="23" rx="117" ry="58"/>
    </g>
  </g>
</svg>
```

**Comment by PeteSchuster** [comment]
so you removed the linear gradient and used CSS instead, then added `preserveAspectRatio="none"`

File: assets/sass/all-digital/base/_defaults.scss [hunk 1]
```diff
@@ -28,7 +28,8 @@ body {
...
+    background-color: linear-gradient(to bottom,  #0066a0 0%,#076699 50%,#003a74 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```
Comment (PeteSchuster) [inline assets/sass/all-digital/base/_defaults.scss:31]: I'm pretty sure gradients are only supported on the background shorthand or background-image rules

**Comment by PeteSchuster** [comment]
looks like you updated charset too

File: assets/sass/all-digital/base/_defaults.scss (hunk 1 above)
Comment (PeteSchuster) [reply assets/sass/all-digital/base/_defaults.scss:31]: also the laying for background images, the "image" you want on top goes first, so:
```
background-image: url('data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20preserveAspectRatio%3D%22none%22%3E%0A%20%20%3Cdefs%3E%0A%20%20%20%20%3Cfilter%20id%3D%22b%22%20color-interpolation-filters%3D%22sRGB%22%3E%0A%20%20%20%20%20%20%3CfeGaussianBlur%20stdDeviation%3D%2255%22%2F%3E%0A%20%20%20%20%3C%2Ffilter%3E%0A%20%20%20%20%3ClinearGradient%20id%3D%22a%22%20gradientUnits%3D%22userSpaceOnUse%22%20x1%3D%22550%22%20y1%3D%224%22%20x2%3D%22550%22%20y2%3D%22343%22%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%220%22%20stop-color%3D%22%230066A0%22%2F%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%22.5%22%20stop-color%3D%22%23076699%22%2F%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%221%22%20stop-color%3D%22%23003A74%22%2F%3E%0A%20%20%20%20%3C%2FlinearGradient%3E%0A%20%20%3C%2Fdefs%3E%0A%20%20%3Cg%20filter%3D%22url%28%23b%29%22%3E%0A%20%20%20%20%3Cg%20fill%3D%22%230E74A0%22%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%2272%22%20cy%3D%22121%22%20rx%3D%2295%22%20ry%3D%2249%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22781%22%20cy%3D%22171%22%20rx%3D%22132%22%20ry%3D%22166%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%221041%22%20cy%3D%22231%22%20rx%3D%22135%22%20ry%3D%22280%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22541%22%20cy%3D%22195%22%20rx%3D%22132%22%20ry%3D%22125%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%20%20%3Cg%20fill%3D%22%230057A0%22%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22383%22%20cy%3D%2223%22%20rx%3D%22205%22%20ry%3D%22136%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22278%22%20cy%3D%22270%22%20rx%3D%2283%22%20ry%3D%2232%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%221083%22%20cy%3D%2223%22%20rx%3D%22117%22%20ry%3D%2258%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%3C%2Fg%3E%0A%3C%2Fsvg%3E'),
    linear-gradient(to bottom, #0066a0 0%, #076699 50%, #003a74 100%);
```

**Comment by PeteSchuster** [comment]
no-scale
the svg also doesn't work with `background-size: cover` as I would expect either. I think viewbox may be required for that.

**Comment by PeteSchuster** [comment]
the original still performs the best on how it "should" scale:
```
background-image: url('data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%221100%22%20height%3D%22343%22%3E%3Cdefs%3E%3Cfilter%20id%3D%22b%22%20color-interpolation-filters%3D%22sRGB%22%3E%3CfeGaussianBlur%20stdDeviation%3D%2255%22%2F%3E%3C%2Ffilter%3E%3ClinearGradient%20id%3D%22a%22%20gradientUnits%3D%22userSpaceOnUse%22%20x1%3D%22550%22%20y1%3D%224%22%20x2%3D%22550%22%20y2%3D%22343%22%3E%3Cstop%20offset%3D%220%22%20stop-color%3D%22%230066A0%22%2F%3E%3Cstop%20offset%3D%22.5%22%20stop-color%3D%22%23076699%22%2F%3E%3Cstop%20offset%3D%221%22%20stop-color%3D%22%23003A74%22%2F%3E%3C%2FlinearGradient%3E%3C%2Fdefs%3E%3Cpath%20fill%3D%22url(%23a)%22%20d%3D%22M0%200h1100v343H0z%22%2F%3E%3Cg%20filter%3D%22url(%23b)%22%3E%3Cg%20fill%3D%22%230E74A0%22%3E%3Cellipse%20cx%3D%2272%22%20cy%3D%22121%22%20rx%3D%2295%22%20ry%3D%2249%22%2F%3E%3Cellipse%20cx%3D%22781%22%20cy%3D%22171%22%20rx%3D%22132%22%20ry%3D%22166%22%2F%3E%3Cellipse%20cx%3D%221041%22%20cy%3D%22231%22%20rx%3D%22135%22%20ry%3D%22160%22%2F%3E%3Cellipse%20cx%3D%22541%22%20cy%3D%22195%22%20rx%3D%22132%22%20ry%3D%22125%22%2F%3E%3C%2Fg%3E%3Cg%20fill%3D%22%230057A0%22%3E%3Cellipse%20cx%3D%22383%22%20cy%3D%2223%22%20rx%3D%22205%22%20ry%3D%22136%22%2F%3E%3Cellipse%20cx%3D%22278%22%20cy%3D%22270%22%20rx%3D%2283%22%20ry%3D%2232%22%2F%3E%3Cellipse%20cx%3D%221083%22%20cy%3D%2223%22%20rx%3D%22117%22%20ry%3D%2258%22%2F%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E');
```

**Comment by PeteSchuster** [comment]
I like the idea of separating the gradient from the SVG, but based on some experiments, its possible that the svg and the gradient can become out of sync which led to some issues as well. I think its best to keep the gradient within the SVG.

**Review by PeteSchuster — CHANGES_REQUESTED** [review]
solution still alludes us

**Comment by PeteSchuster** [comment]
no-scale-2

**Comment by pbcomm** [comment]
Since the project I was implementing this on now falls under a different group, I will let them sort this out.

**Pull request closed without merging**